kong/db/strategies/postgres/connector.lua

```diff
@@ -314,55 +322,94 @@ function _mt:init()
 end
 
 
-function _mt:init_worker(strategies)
-  if ngx.worker.id() == 0 then
+local function cleanup_expired_rows_in_table(config, table_name)
+  -- Create new connection on each table to avoid reusing a connection that might be already timed out
+  local connector = connect(config)
+
+  local cleanup_start_time = now_updated()
+  local ttl_escaped = connector:escape_identifier("ttl")
```

Comment:
Would it be better if we define `ttl_escaped` as a global variable so that we won't have to run escape_identifier every time for the same result.
By the way, personally I don't think either `ttl` or `expired_at` need to be escaped, neither of them is a keyword in current postgres-sql syntax.

Reply:
The `escape_identifier` depends on creating a Postgres connector in pgmoon, and it is merely just string manipulation  so the cost is small, so I think it's okay to keep the current state